### PR TITLE
Merge btc010

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

**Comment by evan82** [comment]
Great Work!

File: contrib/seeds/makeseeds.py [hunk 1]
```diff
@@ -1,32 +1,118 @@
...
 #
 
-NSEEDS=600
+NSEEDS=512
+
+MAX_SEEDS_PER_ASN=2
+
+MIN_BLOCKS = 337600
+
+# These are hosts that have been observed to be behaving strangely (e.g.
```
Comment (rebroad) [inline contrib/seeds/makeseeds.py]: observed by whom?

Comment (schinzelh) [inline contrib/seeds/makeseeds.py]: This is a merge of a Bitcoin commit, so the host list may be inherited from Bitcoins network as noone redacted it.

Comment (thelazier) [inline contrib/seeds/makeseeds.py]: I guess we have this in someone's TODO list already.
https://github.com/dashpay/dash/blob/v0.12.1.x/src/chainparamsseeds.h#L10

**Comment by schinzelh** [comment]
Sure, I'll send a invite to our Developer Slack.

**Pull request merged**